README.md

```diff
@@ -51,6 +51,20 @@ local-packages
 gradle run --args="integ-test -t=EC2Test --package-version={the version you want to test}"
 ````
 
+### Run ECS Integ-test with EC2 on Sidecar mode (EMF Metrics)
+```
+gradle run --args="integ-test -t=ECS_SIDECAR --package-version={the version you want to test} -e launchType=EC2"
```

Comment:
ECS_SIDECAR vs EC2Test, we may want to use all-uppercase for the test case name, it will be great if you can change EC2Test to EC2_TEST :)

Reply:
Yes, Since these are constants enum, I updated them to all upper case with "_".

Reply:
can we build these naming standards into checkstyle?